views/layout.ejs

```diff
@@ -39,8 +39,13 @@
   </head>
 
   <body>
-    
-    <div ng-view=""></div>
+
+      <div class="col-lg-3">
```

Comment:
In order to use Bootstraps grid system, your columns should be inside a `.row`. [reference](http://getbootstrap.com/2.3.2/scaffolding.html#gridSystem). Please include these columns inside a `.row` wrapper